metron-platform/metron-common/src/main/java/org/apache/metron/common/dsl/functions/resolver/ClasspathFunctionResolver.java

```diff
@@ -0,0 +1,180 @@
+ * part of Apache Metron.
+ *
+ *   stellar.function.resolver.includes = org.apache.metron.*
+ *
+ * The following property definition will exclude Stellar functions that are part of
+ * Metron's management suite of function.
+ *
+ *   stellar.function.resolver.excludes = org.apache.metron.management.*
+ *
+ * The following property definition would also exclude the Stellar functions that are
+ * part of the management suite of functions.  Of course, this may also exclude other
+ * packages, but this serves as an example of the types of expression that can be used.
+ *
+ *   stellar.function.resolver.excludes = org\\.management.*
+ *
+ */
+public class ClasspathFunctionResolver extends BaseFunctionResolver {
+
+  /**
+   * The key for a global property that defines one or more regular expressions
+   * that specify what should be included when searching for Stellar functions.
+   */
+  public static final String STELLAR_SEARCH_INCLUDES_KEY = "stellar.function.resolver.includes";
+
+  /**
+   * The key for a global property that defines one or more regular expressions
+   * that specify what should be excluded when searching for Stellar functions.
+   */
+  public static final String STELLAR_SEARCH_EXCLUDES_KEY = "stellar.function.resolver.excludes";
+
+  /**
+   * The includes and excludes can include a list of multiple includes or excludes that
+   * are delimited by these values.
+   */
+  protected static final String STELLAR_SEARCH_DELIMS = "[,:]";
+
+  /**
+   * Regular expressions defining packages that should be included in the Stellar function resolution
+   * process.
+   */
+  private List<String> includes;
+
+  /**
+   * Regular expressions defining packages that should be excluded from the Stellar function resolution
+   * process.
+   */
+  private List<String> excludes;
+
+  public ClasspathFunctionResolver() {
+    this.includes = new ArrayList<>();
+    this.excludes = new ArrayList<>();
+  }
+
+  /**
+   * Includes one or more packages in the Stellar function resolution process.  The packages
+   * to include can be specified with a regular expression.
+   * @param toInclude The regular expressions.
+   */
+  public void include(String... toInclude) {
+    for(String incl : toInclude) {
+      includes.add(incl);
+    }
+  }
+
+  /**
+   * Excludes one or more packages from the Stellar function resolution process.  The packages
+   * to exclude can be specified with a regular expression.
+   * @param toExclude The regular expressions defining packages that should be excluded.
+   */
+  public void exclude(String... toExclude) {
+    for(String excl : toExclude) {
+      excludes.add(excl);
+    }
+  }
+
+  @Override
+  public void initialize(Context context) {
+    super.initialize(context);
+    if(context != null) {
+
+      Optional<Object> optional = context.getCapability(STELLAR_CONFIG, false);
+      if (optional.isPresent()) {
+        Map<String, String> stellarConfig = (Map<String, String>) optional.get();
```

Comment:
Sure.  Done.

At some point, it might be nice to define the type as part of the capability rather than assuming what the type is when you use it.